src/ng-s3upload/ng-s3upload.js

```diff
@@ -2,21 +2,7 @@
 // and are loaded in the correct order to satisfy dependency injection
 // before all nested files are concatenated by Grunt
 
```

Comment:
It would be better to declare of all of the modules here (```angular.module('x', [])```) and remove the duplicates from the specific files. that way you're not depended on loading orders and have all of the dependencies declared in one place.